.travis.yml

```diff
@@ -1,11 +1,23 @@
 language: python
 python:
-  - "2.7"
-  - "3.6"
-  - "3.7"
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
 install:
   - make install
 script:
   make
-dist: xenial
-sudo: true
\ No newline at end of file
+
+matrix:
+  include:
+    # workaround for https://github.com/travis-ci/travis-ci/issues/9815
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+    # black requires 3.6
+    - name: "Black formatting"
+      python: 3.6
```

Comment:
Can't this be latest (3.7)?

Reply:
I suppose it can, but I didn't want to rely on 3.7 since you need the Ubuntu hack to get it working. I can make it 3.7 if you prefer.